api/turing/config/config.go

```diff
@@ -337,7 +340,10 @@ type OpenapiConfig struct {
 	// SpecFile specifies the file path containing OpenAPI v3 spec
 	SpecFile string
 	// Optional. Defines a configuration to be used for serving Swagger UI as a single-page app
+	// If this is provided, it will use the bundled yaml instead.
```

Comment:
I'm not sure if I can follow the logic here. Your PR assumingly makes it possible to path OAS3 spec file and use it for both API validation and serving Swagger UI. 

And with that, I'm not sure that I understand this comment clearly. `SwaggerUIConfig` was expected to provide the information on how to serve Swagger UI and it should work the same with or without patching of the spec

Reply:
> I'm not sure if I can follow the logic here. Your PR assumingly makes it possible to path OAS3 spec file and use it for both API validation and serving Swagger UI.

Not sure if I get what you mean but this is true, you can use both the same file for OpenAPI validation + serving Swagger UI.

> And with that, I'm not sure that I understand this comment clearly. SwaggerUIConfig was expected to provide the information on how to serve Swagger UI and it should work the same with or without patching of the spec

This is also true. You can choose to patch or not patch it and it will still work.